infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuInterClusterReplicationServer.java

```diff
@@ -47,8 +48,9 @@
                     + "\tlog_replication_server (-l <path>|-m) [-nsN] [-a <address>|-q <interface-name>] "
                     + "[--max-num-snapshot-msg-per-batch=<batch-size>] "
                     + "[--max-data-message-size=<msg-size>] "
-                    + "[--lock-lease=<lease-duration>]"
                     + "[-c <ratio>] [-d <level>] [-p <seconds>] "
+                    + "[--node-id=<nodeId>]"
+                    + "[--log-replication-config=<log-replication-config-file-path>]"
```

Comment:
Missing the descriptions for these two fields?

Reply:
Added